trk/src/Screens/GlossaryDefinition/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from "react";
+import { Text,  View, ScrollView, SafeAreaView, StyleSheet, Image } from "react-native";
+import DescriptorsApi from "../../api/DescriptorsApi";
+import storage from '@react-native-firebase/storage';
+
+const GlossaryDefinition = ({ route }) => {
+
+  const { descriptor } = route.params;
+  const [definition, setDefinition] = useState('');
+  const [photoUrl, setPhotoUrl] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const api = DescriptorsApi();
+      try {
+        const querySnapshot = await api.getDescriptorsBySomeField('descriptor', descriptor);
+        if (!querySnapshot.empty) {
+          const doc = querySnapshot.docs[0];
+          const data = doc.data();
+          setDefinition(data.definition);
+
+          // Fetching the image URL from Firebase Storage
+          const photoRef = data.photo ? storage().ref(`${data.photo}`) : storage().ref('default/path/to/image.png');
+          photoRef.getDownloadURL()
+            .then((url) => {
+              setPhotoUrl(url);
+            })
+            .catch((error) => {
+              console.error("Error getting photo URL: ", error);
+            });
+        } else {
+          console.log('No matching document found');
+        }
+      } catch (error) {
+        console.error('Error fetching descriptor:', error);
+      }
+    };
```

Comment:
self explanatory but we fetch the string of the photo (which is a storage pathway)